db/schema.rb

```diff
@@ -75,7 +75,7 @@
     t.string "zip"
     t.string "email"
     t.string "password_digest"
-    t.integer "role"
+    t.integer "role", default: 0
```

Comment:
Woohoo, that's teamwork!

Reply:
![](https://media.giphy.com/media/WP2MUeauAM3sLbOQyQ/giphy.gif)